src/server/CookingApp/Controllers/CostCalculatorController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace CookingApp.Controllers
+{
+    using CookingApp.Services.CostCalculation;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Options;
+    using MongoDB.Bson;
+    using System.Text.Json;
+
+    [ApiController]
+    [AllowAnonymous]
+    [Route("[controller]")]
+    public class CostCalculatorController(ICostCalculationService costService) : ControllerBase
+    {
+        [HttpGet("azure-prices")]
+        public async Task<IActionResult> CalculateAzureServices()
+        {
+            await costService.AzureServices();
+
+            return Ok();
```

Comment:
Firstly in the current project all of the controllers should return only ApiResponse<T> like so:
```cs
return new ApiResponse<Feedback>()
    {
        Status = 200,
        Data = createdFeedback
    };
```
Secondly it is better to always return some information. Here the name "CalculateAzureServices" suggests that information will be returned about the calculated cost of the azure services. So there is either a mistake in the name or in the output.